### ISSUE TITLE
Responsive Design

### ISSUE THREAD

**Issue by jvsinghk** [jvsinghk/spacex#6]
Responsive Behavior:

Implementation should follow a Mobile-first design approach

1. Mobile View: The page should have only one Column until 700 px. Cards take the full width on the screen.

2. Tablet View: The page should have 2 columns of launch details information between 700 and 1024 px. Including the Filter column, it should have 3 Columns on the page.

3. Desktop View: The page should have 4 columns of launch details information between 1024 and 1440 px. Including the Filter column, it should have 5 columns on the page. Beyond 1440px viewport,  the content will be centered align with a max width of 1440.

https://bit.ly/SpacexResponsiveIssue

**Comment by mousamdhakal** [comment]
@jvsinghk  what would you want to do with filters in the mobile view ? will they stay in top or at the side ?

**Comment by zbarovsky** [comment]
Hi! I'd like to try and tackle this if it's still available?

**Comment by jvsinghk** [comment]
@mousamdhakal Will stay on the top will take the first column full width on mobile. Then Space details.

**Comment by mousamdhakal** [comment]
@jvsinghk  Alright, you can assign me this.

**Comment by jvsinghk** [comment]
@zbarovsky Yes, It's available. @mousamdhakal would you like to work on it? Or I assign @zbarovsky

**Comment by zbarovsky** [comment]
he can have it, @mousamdhakal was here first.

**Comment by mousamdhakal** [comment]
Yep , working on it

**Comment by jvsinghk** [comment]
Ok, @mousamdhakal go ahead assigning it to you.

**Comment by jvsinghk** [comment]
@zbarovsky I have added another issue with SSR. Let me know if you would like to contribute to it. Or you can suggest any other issues.

**Comment by jvsinghk** [comment]
@mousamdhakal I have added the review for the #9.

**Comment by mousamdhakal** [comment]
> @mousamdhakal I have added the review for the #9.

I need permission on drive to view mockup, have requested access for the same

**Comment by jvsinghk** [comment]
Check now.

**Comment by jvsinghk** [comment]
@mousamdhakal Did you got it? Let me know if you have any question, need any help or resources to fix this issue. Always happy to help. Happy Hacking!

**Comment by mousamdhakal** [comment]
@jvsinghk  I will work on it later in the day.

**Comment by jvsinghk** [comment]
@zbarovsky This issue is still open. Let me know if you are would like to contribute to this.
